doc/application/application.rst

```diff
@@ -831,44 +834,49 @@ Application Configuration
 Kconfig Configuration
 =====================
 
-The application is configured using a set of options that can be customized for
-application-specific purposes. The Zephyr build system takes a configuration
-option's value from the first source in which it is specified, taken from the
-following available sources, in order:
+The initial configuration for an application is produced by merging
```

Comment:
Can omit initial, it is produced-by-merging after the initial config file is created.